solidity/contracts/token/TokenHolder.sol

```diff
@@ -24,59 +24,36 @@ contract TokenHolder is ITokenHolder, Owned, Utils {
      * @dev withdraws funds held by the contract and sends them to an account
      * can only be called by the owner
      *
-     * @param token ERC20 token contract address (with a special handling of NATIVE_TOKEN_ADDRESS)
+     * @param reserveToken reserve token contract address (with a special handling of NATIVE_TOKEN_ADDRESS)
```

Comment:
We probably wanna remove `NATIVE_TOKEN_ADDRESS` from our official (natspec) documentation.

Reply:
Good point